crates/activate/src/snapshots/activate__test__update.snap

```diff
@@ -228,7 +228,7 @@ expression: "(partition_changes, task_changes)"
               },
               {
                 "name": "estuary.dev/logs-journal",
-                "value": "ops/logs/name"
+                "value": "ops/tasks/BASE_NAME/logs/2020202020202020/kind=derivation/name=example%2Fderivation/pivot=00"
```

Comment:
Just want to double check, but do we still have a test covering the case where the existing journal/shard ids don't have the generation id suffix? I thought perhaps this was the snapshot for that

Reply:
Yes, it's create_and_delete.snap. This is unrelated -- it's replacing an ad-hoc fixture with the correct implied logs partition.